src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -268,3 +268,33 @@
     :image             :icon-avatar
     :image-props       {:icon (status.resources/get-service-image :latamex)}
     :on-press          #(rn/open-url "https://latamex.com")}])
+
+(def networks-list
```

Comment:
from thinking about this, I think we need to remove this temp networks list and instead use the sub I provided. As I understand we have no intention to work with polygon or zksync just yet. @ulisesmac discussed this with design team recently as far as I recall. 
With regards to specific network configuration we can later adjust this properly so that each account has a filtered set of chosen networks from eth opt and arb 👍